src/translations/en.json

```diff
@@ -407,6 +407,44 @@
       }
     },
     "dialogs": {
+      "quick_open": {
+        "commands": {
+          "heading": "YAML configuration reloading",
+          "introduction": "Some parts of Home Assistant can reload without requiring a restart. Hitting reload will unload their current YAML configuration and load the new one.",
+          "reload": "Reload {domain}",
+          "core": "Reload location & customizations",
+          "group": "Reload groups, group entities, and notify services",
+          "automation": "Reload automations",
+          "script": "Reload scripts",
+          "scene": "Reload scenes",
+          "person": "Reload persons",
+          "zone": "Reload zones",
+          "input_boolean": "Reload input booleans",
```

Comment:
We already have all these strings in the config panel too

Reply:
Yeah, the problem was that those strings don't load until you visit the config panel. I copied and pasted here as a workaround until I got answers in Discord.

I'll be moving the strings out of config bucket and into "dialogs" with refs

Reply:
It might be better to move this into its own section.

As `dialogs` will always be loaded for every panel and I can see this list grow pretty fast.

Reply:
> As dialogs will always be loaded for every panel and I can see this list grow pretty fast.

I don't think `dialogs` loads on every panel. It comes through in the global translations on first page load. Navigating to other panels only loads in the strings related to that panel.

Reply:
yes, that is what I mean, no matter on what panel you are, it will be loaded. Meaning it will slow the page load down, also for users that will not use this (as this is a power user feature).

Reply:
Hm. Maybe I'm missing something. Maybe the way I'm verifying this is flawed:

1. Launch HA in browser and open dev tools
2. Go to "Network" tab and filter for "en-dev.json" to watch for translation fetches
3. Hard refresh the page
4. A call to http://localhost:8123/static/translations/en-dev.json is made which returns all the quick-open strings
5. Navigate to another panel (say, Configuration)
6. A call to http://localhost:8123/static/translations/config/en-dev.json is made which does NOT include the quick-open strings
7. Repeat 5-7 for any other panels. I haven't seen the quick-open strings being fetched for any other panels.

Are you talking about something other than an api fetch? Like, loading stuff into local storage or something?

Reply:
No, but I don't want it in the main translations package. I want it to be lazy loaded

Reply:
Okay. How's `ui.quick_open.*` sound

Reply:
It was tricky, but I managed to get the quick open strings to:

* Not load in the main package
  * A gulp task takes a list of exclusion keys, and splits them into chunks. But the key has to be in ui.panel, so I had to move the strings there.
* "Lazy load" strings without changing URL
  * We only lazy load strings when changing URL and re-rendering HassElement which uses its TranslationMixin to fetch and then store panel-related strings locally. But I was able to use `litLocalizeLiteMixin` to load strings after rendering the dialog.

Reply:
OK, sorry, but let's not go down that path for now. I think it would be best to leave it in dialog then and we can change it later if needed.

Reply:
Agreed. And learned a ton along the way

Reply:
Reverted